include/ultra64/libc.h

```diff
@@ -0,0 +1,12 @@
+#ifndef ULTRA64_LIBC_H
+#define ULTRA64_LIBC_H
+
+#include "stddef.h"
+
+void osSyncPrintf(const char* fmt, ...);
+
+void bzero(void* __s, size_t __n);
+int bcmp(const void* __sl, const void* __s2, size_t __n);
+void bcopy(const void* __src, void* __dest, size_t __n);
```

Comment:
The original libultra headers use `int` instead of `size_t`. Why deviate from the original headers here?
Also, why the `__` prefix on the arguments?

Reply:
IIRC `int` is required for matching anyway. Currently in HEAD it's

```
#ifdef __GNUC__
void bzero(void* __s, unsigned int __n);
int bcmp(const void* __sl, const void* __s2, unsigned int __n);
void bcopy(const void* __src, void* __dest, unsigned int __n);
#else
void bzero(void* __s, int __n);
int bcmp(const void* __sl, const void* __s2, int __n);
void bcopy(const void* __src, void* __dest, int __n);
#endif
```